src/webgpu/shader/execution/memory_model/coherence.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { GPUTest } from '../../../gpu_test.js';
+
+export const description = `
+A suite of tests checking threads all see a sequentially consistent view of the order of memory
```

Comment:
nit: `A suite of tests checking threads all` -> `Tests that all threads`
Not a huge issue, your more concise descriptions earlier read better to me.

Reply:
done!